### ISSUE TITLE
baal helper/diablo helper crashes

### ISSUE THREAD

**Issue by hdikiglb** [kolton/d2bot-with-kolbot#1892]
I checked around the issues and saw a few others with this issue, i tried doing what they did (mainly using the tags/unicode, also tried only using -w in parameters) but with no success, anybody else have a solution? thanks.

example:
11:57:56 () Window has unexpectedly exited... starting profile
11:58:05 () Crash Info: Script: Diablo Area: Chaos Sanctuary
12:11:21 () Window has unexpectedly exited... starting profile
12:11:24 () Crash Info: Script: BaalHelper Area: Throne Of Destruction
12:11:41 () Failed to join game
12:56:00 ()  Error in DiabloHelper (diablohelper.js #227) Diablo not found (Area: 108, Ping:110, Game: )
13:06:28 () Window has unexpectedly exited... starting profile
13:06:31 () Crash Info: Script: Diablo Area: River Of Flame
13:10:52 ()  Error in Endugu (attack.js #435) Attack.clear: Witch Doctor Endugu not found (Area: 91, Ping:562, Game: )
13:15:08 () Window has unexpectedly exited... starting profile
13:15:16 () Crash Info: Script: Diablo Area: Chaos Sanctuary
14:06:29 () Window has unexpectedly exited... starting profile
14:06:33 () Crash Info: Script: Diablo Area: Chaos Sanctuary
14:35:45 () Window has unexpectedly exited... starting profile
14:35:48 () Crash Info: Script: BaalHelper Area: Throne Of Destruction
14:36:05 () Failed to join game
14:50:46 ()  Error in DiabloHelper (diablohelper.js #227) Diablo not found (Area: 108, Ping:203, Game: )
14:52:04 () Window has unexpectedly exited... starting profile
14:52:09 () Crash Info: Script: no entry Area: Harrogath
14:58:23   Error in Endugu (attack.js #435) Attack.clear: Witch Doctor Endugu not found (Area: 91, Ping:125, Game: )
15:06:28  Window has unexpectedly exited... starting profile
15:06:31  Crash Info: Script: Baal Area: Throne Of Destruction
15:14:04   Error in SharpTooth (attack.js #229) Attack.kill: Target not found (Area: 111, Ping:234, Game: )
16:42:18  Window has unexpectedly exited... starting profile
16:42:21 Crash Info: Script: DiabloHelper Area: Chaos Sanctuary
16:42:38 Failed to join game
17:28:45 ) Window has unexpectedly exited... starting profile
17:28:47  Crash Info: Script: BaalHelper Area: Halls Of Vaught
17:29:05  Failed to join game

**Comment by hdikiglb** [comment]
i guess it's not just on diablo/baal helper it happens randomly during all

https://pastebin.com/1LXL9Pvf

**Comment by hdikiglb** [comment]
this is a fresh install of kolbot as of a month ago, also just reinstalled all 4 of my diablo instances, still crashing.

**Comment by diablodude** [comment]
have you installed the english version of diablo2?

**Comment by hdikiglb** [comment]
> 
> 
> Are you using an English installation of Diablo2?

yeah all instances are english, i just did a fresh install of kolbot/d2bs just to see what happens, once it runs a little i'll post an update

**Comment by hdikiglb** [comment]
nobody have anything to add?  have no idea where to go from here, if you need any log files or anything lmk

**Comment by hdikiglb** [comment]
anyone check these anymore? still broken

**Comment by hdikiglb** [comment]
re downloaded yet again and using only unicode as said to use on the discord, same issue.
